Signal/src/ViewControllers/ConversationView/ConversationInputTextView.m

```diff
@@ -56,6 +56,7 @@ - (instancetype)init
         self.placeholderView.text = NSLocalizedString(@"new_message", @"");
         self.placeholderView.textColor = [UIColor lightGrayColor];
         self.placeholderView.userInteractionEnabled = NO;
+        self.placeholderView.textAlignment = NSTextAlignmentNatural;
```

Comment:
Isn't this default? Why do we need to specify this?

Reply:
Removed.